### ISSUE TITLE
Feature Request: Can we do something like this with some box drawing characters?

### ISSUE THREAD

**Issue by SATYADAHAL** [emacs-dashboard/emacs-dashboard#535]
![Screenshot_20240310_172424](https://github.com/emacs-dashboard/emacs-dashboard/assets/51309409/35e53a2a-668b-49e8-9565-8c273d561227)
![Screenshot_20240310_174322](https://github.com/emacs-dashboard/emacs-dashboard/assets/51309409/525a21ce-28ec-45d2-a5de-24d4b0941c72)

**Comment by ricardoricho** [comment]
Yes, looks nice to have it like an option. Do you have some code?

**Comment by SATYADAHAL** [comment]
Unfortunately, I don't have the code for this  at the moment as I'm not very experienced with Emacs Lisp. However, I'm happy to collaborate or provide further details to assist in the development.
